### DISCUSSION TITLE
Can I have definition popup like VSCode?

### DISCUSSION THREAD

**Discussion by ninthsun91** [zed-industries/zed#11598] [Feature Requests]
In VSCode, when I open parentheses to call a function, VSCode will show a popup with the function's definition like this:
![image](https://github.com/zed-industries/zed/assets/50778955/f1806fc1-7f6e-4e2e-a8af-044f1c22b460)
![image](https://github.com/zed-industries/zed/assets/50778955/41e91605-7dcf-4e75-89b7-655d0b603e5d)

I feel this tooltip very useful, but I can't quite find a relevant configuration in Zed.
Is there a way to enable this function definition tooltip box?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
